packages/cli/src/fingerprint/fingerprintWatchCommand.js

```diff
@@ -1,4 +1,6 @@
 const chokidar = require('chokidar');
+const fs = require('fs-extra');
+const path = require('node:path');
```

Comment:
I think Dustin said we should use the old `path` instead of `node:path`? @dustinbyrne ?